src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItemTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.config;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ServiceConfigItemTest {
```

Comment:
looks like this only contains tests for form type to surname field config. Rename please

Reply:
Will rename to ServiceConfigItemFormFieldMappingsTest